### PR TITLE
Adding country selector

### PR DESCRIPTION
**Pull request by zobione**

### CODE REVIEW

File: app/angular/src/core/location/AddPostalCodeList.html [hunk 1]
```diff
@@ -5,17 +5,537 @@
...
+        </div>
+        <div class="radio">
+          <label>
+            <input type="radio" name="optionsRadios" id="optionsRadios2" ng-model="input.type" value="country">
+              Target by country
+          </label>
+        </div>
+      <div mcs-form-group="select" label-for="country" label-text="Country" ng-show="input.type === 'zipcode'">
           <select class="form-control"  id="country" ng-model="input.country" required="true">
+              <option value="AF">Afghanistan</option>
```
Comment (dduponchel) [inline app/angular/src/core/location/AddPostalCodeList.html]: Don't hard code (twice) the list of countries, use the API, **or** use a small subset of interesting countries

File: app/angular/src/core/location/AddPostalCodeListController.js [hunk 2]
```diff
@@ -21,16 +35,30 @@ define(['./module'], function (module) {
...
-            postal_code : postalCodeAdded
-          });
+        
+        if ($scope.addedCountries.length !==0 && $scope.input.type==='country') {
+           var addedCountries;
+           for (var j = 0; j < $scope.addedCountries.length; j++) {
+              addedCountries = $scope.addedCountries[j];
+              $scope.$emit("mics-location:country-added", {
+                id: IdGenerator.getId(),
+                type: 'GEOCODE',
```
Comment (dduponchel) [inline app/angular/src/core/location/AddPostalCodeListController.js]: `GEONAME`, not `GEOCODE`

File: app/angular/src/core/location/AddPostalCodeList.html [hunk 3]
```diff
@@ -5,17 +5,537 @@
 
   <div class="modal-body">
       <form>
-      <div mcs-form-group="select" label-for="country" label-text="Country">
+        <label for="targetingSelection">Test</label>
```
Comment (dduponchel) [inline app/angular/src/core/location/AddPostalCodeList.html]: "Test" ?

File: app/angular/src/core/location/AddPostalCodeList.html [hunk 4]
```diff
@@ -5,17 +5,537 @@
...
+              <option value="WF">Wallis and Futuna</option>
+              <option value="EH">Western Sahara</option>
+              <option value="YE">Yemen</option>
+              <option value="ZM">Zambia</option>
+              <option value="ZW">Zimbabwe</option>
+          </select>
+      </div>
+      <div class="" ng-hide="addedCountries.length === 0">
+        <label>Selected Countries:</label>
+        <p class="" ng-repeat="country in addedCountries track by $index"><button ng-click="removeCountry(country)" type="button" class="close pull-right" aria-label="Close"><span aria-hidden="true">&times;</span></button>{{ country }}</p>
```
Comment (zobione) [inline app/angular/src/core/location/AddPostalCodeList.html]: Understood but it allow us to do kind of a multiselect

**Pull request merged**